packages/commerce-sdk-react/src/auth/index.ts

```diff
@@ -403,9 +411,11 @@ class Auth {
         this.set('customer_type', isGuest ? 'guest' : 'registered')
 
         const refreshTokenKey = isGuest ? 'refresh_token_guest' : 'refresh_token_registered'
-        const expiresDate = res.refresh_token_expires_in
-            ? this.convertSecondsToDate(res.refresh_token_expires_in)
-            : undefined
+        const fallbackTTL =
+            typeof this.refreshTokenTTL === 'number' && this.refreshTokenTTL >= 0
+                ? this?.refreshTokenTTL
```

Comment:
`this` should always be defined.

```suggestion
                ? this.refreshTokenTTL
```

Reply:
Done ✅